src/components/addressAutocomplete/useSuggestions.tsx

```diff
@@ -17,6 +17,20 @@ type configProps = {
   minCharactersLength?: number;
 };
 
+const debounce = (handler: (...arg: any) => any, timer: number) => {
```

Comment:
why dont you use your service ?

Reply:
It is an work in progress. I have made the service here before extract but this function won't stay here.

Reply:
Why not :-p Sometimes i use this syntax for the uninterestings let/var, but if you prefer, i clean.

Reply:
then don't ask me to review it...